ShopGo/CreateWishListScreen/WishListViewController.swift

```diff
@@ -8,7 +8,10 @@
 import UIKit
 
 class WishListViewController: UIViewController {
-    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setUpWishListButton()
+    }
     var wishListLabelArray: [String] = ["Watch","Heels","Shoes" ,"Shoes", "Bag", "Dress", "Watch", "Bag", "Shoes", "Dress"]
      var wishListMoneyArray: [String] = ["20$"]
```

Comment:
Outlets and variables should come before viewDidLoad then methods

Reply:
fixed